### ISSUE TITLE
Adopt signal-style bubble-styling of several messages of the same user

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/talk-android#2365]
From https://github.com/nextcloud/talk-android/pull/2364#pullrequestreview-1105428929 - adopt chat bubble corner dynamics for multiple message of a single user, i.e. like signal:

![Screenshot_20220913-110951_Signal.jpg](https://user-images.githubusercontent.com/52440189/189862652-3e5efe9f-1b05-4544-9c06-677619980783.jpg)

**Comment by mahibi** [comment]
we already had this (`chatMessage.isGrouped`), seems to be broken right now..

**Comment by mahibi** [comment]
just noticed in passing:

the grouping seems to work fine but it's not handled design-wise (the first message is different, for the following messages the round corners are just too large). i guess this came with the switch to material3 

![grafik](https://user-images.githubusercontent.com/2932790/190095064-d751969e-e1ec-49a2-8960-5402c9f004dc.png)

**Comment by AndyScherzinger** [comment]
The generally larger corners came with the material3 changes, but nothing else, so where is the grouping logic to be found?

**Comment by mahibi** [comment]
Yes you're right material3 didn't change anything regarding the logic. The current behaviour is unchanged since ever, so it's indeed a feature request and not a bug.

We have the same behaviour like whatsapp. The first bubble is a "speech bubble" and the following ones are just rounded.

> so where is the grouping logic to be found?

TalkSpecificViewThemeUtils#themeIncomingMessageBubble
TalkSpecificViewThemeUtils#themeOutgoingMessageBubble

**Comment by mahibi** [comment]
signal | telegram  
------- | -------- 
![grafik](https://user-images.githubusercontent.com/2932790/190341794-222e97b0-0b2b-4412-94e8-9b4fb081396a.png) | ![grafik](https://user-images.githubusercontent.com/2932790/190341904-3f9d7fda-3b17-44e7-8869-c2ff0c0ecfa9.png) 

whatsapp | talk 
-------- | -------- 
![grafik](https://user-images.githubusercontent.com/2932790/190341991-bbc50dbe-2fa3-408b-a151-d415bb55eddf.png) | ![grafik](https://user-images.githubusercontent.com/2932790/190342592-a8ca6a41-4d47-4a70-8030-650ce929c039.png)

**Comment by AndyScherzinger** [comment]
Superseded by #5540, which tracks aligning the bubble style with the new layout from server and iOS. Closing in favour of #5540.
